starlette_feedgen/feed.py

```diff
@@ -22,6 +21,11 @@ class FeedEndpoint(HTTPEndpoint, ABC):
     domain: Optional[str] = None
     link: str = "/"
 
+    def __init__(self, *args, **kwargs) -> None:
+        super().__init__(*args, **kwargs)
+        self.use_cached_items = False
+        self.cached_items = []
+
     @abstractmethod
     async def get_items(self) -> Iterable:
```

Comment:
You should change the annotation of this method to `Union[Iterable, AsyncIterable]`

Reply:
agree